hedera-node/src/test/java/com/hedera/services/queries/schedule/GetScheduleInfoAnswerTest.java

```diff
@@ -0,0 +1,243 @@
+package com.hedera.services.queries.schedule;
+
+import com.hedera.services.context.primitives.StateView;
+import com.hedera.services.txns.validation.OptionValidator;
+import com.hederahashgraph.api.proto.java.AccountID;
+import com.hederahashgraph.api.proto.java.HederaFunctionality;
+import com.hederahashgraph.api.proto.java.Query;
+import com.hederahashgraph.api.proto.java.QueryHeader;
+import com.hederahashgraph.api.proto.java.Response;
+import com.hederahashgraph.api.proto.java.ResponseCodeEnum;
+import com.hederahashgraph.api.proto.java.ResponseType;
+import com.hederahashgraph.api.proto.java.ScheduleGetInfoQuery;
+import com.hederahashgraph.api.proto.java.ScheduleGetInfoResponse;
+import com.hederahashgraph.api.proto.java.ScheduleID;
+import com.hederahashgraph.api.proto.java.ScheduleInfo;
+import com.hederahashgraph.api.proto.java.Transaction;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import static com.hedera.test.factories.scenarios.TxnHandlingScenario.COMPLEX_KEY_ACCOUNT_KT;
+import static com.hedera.test.utils.IdUtils.asAccount;
+import static com.hedera.test.utils.IdUtils.asSchedule;
+import static com.hedera.test.utils.TxnUtils.payerSponsoredTransfer;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_SCHEDULE_ID;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.OK;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.RESULT_SIZE_LIMIT_EXCEEDED;
+import static com.hederahashgraph.api.proto.java.ResponseType.ANSWER_ONLY;
+import static com.hederahashgraph.api.proto.java.ResponseType.COST_ANSWER;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+public class GetScheduleInfoAnswerTest {
+    private Transaction paymentTxn;
+    private String node = "0.0.3";
+    private String payer = "0.0.12345";
+    private AccountID payerAccount = asAccount(payer);
+    private AccountID schedulingAccount = asAccount("0.0.12346");
+    private ScheduleID scheduleID = asSchedule("1.2.3");
+    private long fee = 1_234L;
+
+    StateView view;
+    OptionValidator optionValidator;
+
+    ScheduleInfo info;
+
+    GetScheduleInfoAnswer subject;
+
+    @BeforeEach
+    public void setup() {
+        info = ScheduleInfo.newBuilder()
+                .setScheduleID(scheduleID)
```

Comment:
I get a compile error on this line, which makes sense given the `schedule` name for the `ScheduleID` field in [_ScheduleGetInfo.proto_](https://github.com/hashgraph/hedera-services/blob/scheduled-transactions/grpc-protobuffers-base/hapi-proto/src/main/proto/ScheduleGetInfo.proto#L40).